docs/jwst/firstframe/arguments.rst

```diff
@@ -1,4 +1,8 @@
 Step Arguments
 ==============
 
-The first frame correction has no step-specific arguments.
+The ``firstframe`` step has the following step-specific arguments.
+
+``--bright_use_group1`` (boolean, default=True)
+    If True, for pixels that have the saturation flag set for group 3, 
+    but not group 2 will not have their group 1 groupdq set to DO_NOT_USE.
```

Comment:
Suggestion (some like): 
If True, for pixels that have the saturation flag set for group 3,  but not group 2  then the groupdq for group 1 will not be set DO_NOT_USE. This will allow a slope to be determined for this pixel.

Reply:
I've reworded motivated by your comment.  Hopefully the new wording works.